src/app/public/modules/confirm/confirm-config.ts

```diff
@@ -7,9 +7,28 @@ import {
 } from './confirm-button-config';
 
 export interface SkyConfirmConfig {
+  /**
+   * Specifies the message to display in a bold font at the top of the dialog.
+   * @required
+   */
   message: string;
+  /**
+   * Specifies secondary text to display in plain font under the primary message.
+   */
   body?: string;
+  /**
+   * Specifies an array of `SkyConfirmButtonConfig` objects that overwrite the default
+   * configuration for buttons. The number of items in the array must match the number of
+   * buttons that the `type` property specifies.
+   */
   buttons?: SkyConfirmButtonConfig[];
+  /**
+   * Indicates whether to preserve whitespace and new lines inside the confirm component.
+   * @default false
+   */
   preserveWhiteSpace?: boolean;
+  /**
+   * Specifies a SkyConfirmType enum that indicates how many buttons to include in the dialog.
```

Comment:
We're losing the rest of the info with this property in the current docs. I assume that we need to create JSDocs comments for `Custom`, `OK`, `YesCancel`, and `YesNoCancel` in https://github.com/blackbaud/skyux-modals/blob/master/src/app/public/modules/confirm/confirm-type.ts. Right now when I serve the docs, we get a table for those options but not descriptions, so I think we need to copy that over.

Reply:
Added and done!